_data/projects/Appsmith.yml

```diff
@@ -0,0 +1,13 @@
+name: Appsmith
+desc: Build custom CRUD apps that talk to any API or database. All without HTML or CSS. 
+site: https://appsmith.com/
+tags:
+  - low-code
+  - internal-tools
+  - crud-apps
+  - admin-panels
+  - app-builder
+  - admin-dashboard
+upforgrabs:
+  name: Good First Issue
+  link: https://github.com/appsmithorg/appsmith/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22+bug
```

Comment:
@cokoghenun Please use this instead https://github.com/appsmithorg/appsmith/labels/Good%20First%20Issue

Reply:
I've updated this @ritwik12

Reply:
Thank you